tests/coalaCITest.py

```diff
@@ -172,3 +172,19 @@ def test_limit_files_negative(self):
             self.assertFalse(stderr)
             self.assertEqual(retval, 0,
                              'coala must return zero when successful')
+
+    def test_bear_dirs(self):
+        with bear_test_module(), \
+                prepare_file(['random_text'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala', '--non-interactive',
+                    '-c', os.devnull,
+                    '--bear-dir', 'test_bears',
+                    '-b', 'LineCountTestBear',
```

Comment:
I think there should be `LineCountBearTest`.

Reply:
@sangamcse  what do you mean o.O ? Name of the bear is LineCountTestBear ;)

Reply:
Are you sure? Is this https://github.com/coala/coala-bears/blob/master/tests/general/LineCountBearTest.py you are talking about?

Reply:
Cool. Got it. 😉